src/login/logind-button.c

```diff
@@ -413,10 +520,42 @@ static int button_suitable(int fd) {
         return false;
 }
 
+static int button_get_modifier_mask(Button *b) {
```

Comment:
hmm, functions with "get" in the name in our codebase usually return what they are getting. Here this is not done, the acquired info is written to the specified object only.

maybe hence use the word "acquire" or "retrieve" or so instead.

Reply:
Thinking about it, button_**initialize**_modifier_mask probably works